react/components/NotificationsDropdown/components/Notification/components/NotificationObjectLink/index.js

```diff
@@ -25,8 +27,11 @@ const NotificationObjectLink = ({
     >
       <a href={href}>
         <Truncate length={40}>
-          {label}
+          {unescape(label)}
```

Comment:
what's this about?

Reply:
&nbsp;

Reply:
lol `&nbsp;`

Reply:
??

Reply:
Same thing as everything everywhere, escaping channel titles with ampersands. Remember, we decided to handle manually on the various clients.

Reply:
yeah but handling it manually is just using dangerouslySetHTML. so why unescape?

Reply:
will fix

Reply:
i mean this is basically what we were talking about doing on the api; but as we decided there this will unescape < / > which is like maybe a problem

Reply:
kk, will fix.

Reply:
Remembered why I did this here, this truncate component makes it particularly hard to do `dangerouslySetHTML`. Can truncate manually or do this, you pick.

Reply:
Just pull out truncate; there's a fn in there.

Reply:
https://github.com/aredotna/ervell/blob/ea52d7ba084d726385a871ffb5a7558ac860a27e/react/components/UI/Truncate/index.js#L7

`import { truncate } from...`